pilot/pkg/networking/plugin/authz/authorization.go

```diff
@@ -49,9 +48,7 @@ func NewBuilder(actionType ActionType, push *model.PushContext, proxy *model.Pro
 		IsCustomBuilder: actionType == Custom,
 	}
 	policies := push.AuthzPolicies.ListAuthorizationPolicies(proxy.ConfigNamespace, proxy.Labels)
-	if !util.IsIstioVersionGE117(proxy.IstioVersion) {
-		option.UseAuthenticated = true
-	}
+	option.UseAuthenticated = false
```

Comment:
if this is always `false` not sure we need an option here.

Although probably the grpcgen should use NewBuilder...

Reply:
removed UseAuthenticated